src/acquisition/covidcast/database.py

```diff
@@ -248,7 +249,7 @@ def insert_or_update_batch(self, cc_rows, batch_size=2**20, commit_partial=False
 
   def compute_covidcast_meta(self, table_name='covidcast', use_index=True):
     """Compute and return metadata on all non-WIP COVIDcast signals."""
-
+    logger = get_structured_logger("compute_covidcast_meta", log_exceptions=False)
```

Comment:
Oh because I check the previous logging example, I saw when logger use level exception, the value will change by True (the default value) to False, so I think maybe this is the one that need to be fix when I use the exception. That's fine, I will fixed that to keep it as default value.